dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "619GB",
-        "label": "CENTRAL ALABAMA HCS - DOTHAN CBOC"
+        "label": "Dothan 1 VA Clinic"
       },
       {
         "value": "521GG",
-        "label": "BESSEMER CBOC"
+        "label": "Bessemer VA Clinic"
       },
       {
         "value": "619GD",
-        "label": "WIREGRASS CBOC"
+        "label": "Wiregrass VA Clinic"
       },
       {
         "value": "521GI",
-        "label": "GUNTERSVILLE CBOC"
+        "label": "Guntersville VA Clinic"
       },
       {
         "value": "521GH",
-        "label": "CHILDERSBURG CBOC"
+        "label": "Childersburg VA Clinic"
       },
       {
         "value": "679HK",
-        "label": "MOBILE CLINIC TUSCALOOSA"
+        "label": "Tuscaloosa VA Mobile Clinic"
       },
       {
         "value": "619GE",
-        "label": "MONROE COUNTY CBOC"
+        "label": "Monroe County VA Clinic"
       },
       {
         "value": "679GA",
-        "label": "SELMA VA CLINIC"
+        "label": "Selma VA Clinic"
       },
       {
         "value": "521GJ",
-        "label": "BIRMINGHAM VA CLINIC"
+        "label": "Birmingham VA Clinic"
       },
       {
         "value": "521",
-        "label": "BIRMINGHAM VAMC"
+        "label": "Birmingham VA Medical Center"
       },
       {
         "value": "619",
-        "label": "CENTRAL ALABAMA HEALTH CARE SYSTEM - MONTGOMERY DIVISION"
+        "label": "Central Alabama VA Medical Center-Montgomery"
       },
       {
         "value": "679",
-        "label": "TUSCALOOSA VA MEDICAL CENTER"
+        "label": "Tuscaloosa VA Medical Center"
       }
     ],
     "OR": [
       {
         "value": "653BY",
-        "label": "EUGENE, OR CBOC"
+        "label": "Eugene VA Clinic"
       },
       {
         "value": "653GA",
-        "label": "NORTH BEND VA CLINIC"
+        "label": "North Bend VA Clinic"
       },
       {
         "value": "653GB",
-        "label": "BROOKINGS VA CLINIC"
+        "label": "Brookings VA Clinic"
       },
       {
         "value": "648GA",
-        "label": "BEND, OR CBOC"
+        "label": "Bend VA Clinic"
       },
       {
         "value": "648GB",
-        "label": "SALEM VA CLINIC"
+        "label": "Salem VA Clinic"
       },
       {
         "value": "648GD",
-        "label": "NORTH COAST VA CLINIC"
+        "label": "North Coast VA Clinic"
       },
       {
         "value": "692GA",
-        "label": "KLAMATH FALLS VA CLINIC"
+        "label": "Klamath Falls VA Clinic"
       },
       {
         "value": "648GF",
-        "label": "HILLSBORO VA CLINIC"
+        "label": "Hillsboro VA Clinic"
       },
       {
         "value": "692GB",
-        "label": "GRANTS PASS CBOC"
+        "label": "Grants Pass VA Clinic"
       },
       {
         "value": "648GG",
-        "label": "WEST LINN CBOC"
+        "label": "West Linn VA Clinic"
       },
       {
         "value": "648GE",
-        "label": "EAST PORTLAND VA CLINIC"
+        "label": "Fairview VA Clinic"
```

Comment:
Let's confirm this name change.

Reply:
This code (648GE) is referenced as "Fairview" in both the [Facility Locator](https://www.va.gov/find-locations/facility/vha_648GE) and the [Portland Facilities Website](https://www.portland.va.gov/locations/Fairview_Clinic.asp). I don't see any reference to "EAST PORTLAND" though. Regardless, I think it's safe to say we have the right facility here.